Modules/Package.swift

```diff
@@ -25,8 +25,7 @@ let package = Package(
         .package(url: "https://github.com/Automattic/AutomatticAbout-swift", from: "1.1.5"),
         .package(url: "https://github.com/Automattic/Gravatar-SDK-iOS", from: "3.2.0"),
         .package(url: "https://github.com/Automattic/Gridicons-iOS", branch: "develop"),
-        .package(url: "https://github.com/Automattic/ScreenObject", from: "0.2.3"),
-        .package(url: "https://github.com/Automattic/XCUITestHelpers", from: "0.4.0"),
+        .package(url: "https://github.com/Automattic/ScreenObject", revision: "2a3fb0bfc7426858ed907d08c1921660201a8f79"),
```

Comment:
This shall become 

```suggestion
        .package(url: "https://github.com/Automattic/ScreenObject", from: "0.3.0"),
```

once https://github.com/Automattic/ScreenObject/pull/17 lands and a new version ships.